instrumentation/net/http/otelhttp/transport.go

```diff
@@ -41,6 +41,10 @@ var _ http.RoundTripper = &Transport{}
 // NewTransport wraps the provided http.RoundTripper with one that
 // starts a span and injects the span context into the outbound request headers.
 func NewTransport(base http.RoundTripper, opts ...Option) *Transport {
+	if base == nil {
```

Comment:
Can you add documentation of this behavior to the godoc for this function?

Reply:
updated